va/validation-authority.go

```diff
@@ -549,13 +551,26 @@ func (va *ValidationAuthorityImpl) validateChallenge(ctx context.Context, identi
 	}
 }
 
-// UpdateValidations runs the validate() method asynchronously using goroutines.
+// UpdateValidations runs the validate() method asynchronously using
+// goroutines.
+//
+// TODO(#1167): remove this method
 func (va *ValidationAuthorityImpl) UpdateValidations(authz core.Authorization, challengeIndex int) error {
 	// TODO(#1292): add a proper deadline here
-	go va.validate(context.TODO(), authz, challengeIndex)
+	go func(authz core.Authorization, challengeIndex int) {
+		ctx := context.Background()
```

Comment:
I think changing this to `Background()` is not correct. From the docs: "Background is typically used in main, init, and tests, and as the top-level Context for incoming requests." I think Background should properly be used in the RPC layer, and passed into the handlers for RPC calls. I believe when we switch to gRPC it will provide appropriate Contexts for us.